samples/ElmInspiredOne/ElmInspiredOneXF.iOS/AppDelegate.cs

```diff
@@ -0,0 +1,76 @@
+﻿using UIKit;
+using Foundation;
+using Gjallarhorn.Bindable.Framework;
```

Comment:
In the android app, there is a reference to Gjallarhorn.XamarinForms.
I was experimenting with just trying to use Gjallarhorn.Bindable without the XamarinForms...